### ISSUE TITLE
IPTorrents can find movie

### ISSUE THREAD

**Issue by jgautier** [RuudBurger/CouchPotatoServer#4874]
### Steps to reproduce:
1. Setup IPTorrent as provider
2. search for movie

### Information:
Movie(s) I have this with: all
Quality of the movie being searched: 1080p, 720p
Providers I use: iptorrents
Version of CouchPotato: source:(RuudBurger:CouchPotatoServer master) edac023e (2015-04-20 22:42:31)
Running on: freenas

### Logs:
```
04-22 20:28:17  INFO [hpotato.core.plugins.base] Opening url: get https://iptorrents.eu/t?l48=&q=%22apocalypse+now%22+1979&qf=#torrents&p=1, data: [] 
```

No matter what I do I can not get iptorrents to find any releases.  When I open the iptorrents url manually I see results.

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Fill in all the dots. They're there for a reason
- [x] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [x] Please wrap inline logs inside ``` for readability
- [ ] Enabled debug logging in Settings > Advanced (restart CP, be sure to disable after the bug is fixed)
- [x] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by mixednutts** [comment]
I had a similar issue on my NAS and had to install lxml for python.
Have a look at http://lxml.de/installation.html for ways to go about installing it.

**Comment by OzanMakinaci** [comment]
Also getting this issues, same thing in the logs.  No error or anything new when the logs are set to debug mode either.  I am not sure how to try to the lxml solution since my environment is Windows 7.

**Comment by mixednutts** [comment]
@jgautier did lxml resolve your issue or is it something else?  

I am on IPT and current CP master branch and using the same search string except with a few extra qualities CP snatches a torrent:

https://iptorrents.eu/t?l7=&l48=&l20=&q=%22apocalypse+now%22+1979&qf=#torrents&p=1

**Comment by mixednutts** [comment]
@OzanMakinaci found this website for lxml on windows but still not sure if lxml is the issue or not?

http://www.zlatkovic.com/libxml.en.html

**Comment by GonzoMoore** [comment]
I just fixed this a couple of days ago on windows by download lxml 3.4.3 python 2.7 binary pack: https://pypi.python.org/pypi/lxml/3.4.3

Restart CP after the install is done and do your search on IPT again.

**Comment by jgautier** [comment]
I followed this and it fixed the issue.

https://couchpota.to/forum/viewtopic.php?t=4598#p22731

**Comment by jrochet** [comment]
Seriously this issue is in about a million different issue reports and there is never a clear solution.  I want to punch it in the throat.

**Comment by becian** [comment]
@jrochet Hey mate, i found on my synology that changing from master to develop fixed the problem

https://github.com/CouchPotato/CouchPotatoServer/issues/6826

**Comment by jrochet** [comment]
Thanks, I actually went with a different fix, while in the process of trying to figure out how to do what you suggested.  I previously had been using the Windows installer version.  When I went looking for the developer branch, I saw the instructions for running from source.  So I tried the master branch with running from source instead, and it did the trick.  I had already installed lxml on my system previously so that could have been the solution since the Windows installer version comes precanned with its own python and lxml files but didn't seem to be working.
